package.json

```diff
@@ -13,7 +13,7 @@
     "bumpPackages": "node scripts/bumpPackages/cli",
     "bumpChangelogs": "node scripts/bumpChangelogs/cli",
     "regeneratePackageLocks": "node scripts/regeneratePackageLocks",
-    "deploy-storybook": "npm run build:storybook && gh-pages -d storybook_dist",
+    "deploy-storybook": "gh-pages -d storybook_dist",
```

Comment:
Just to clarify, why is the call to `build:storybook` no longer needed here? Because it's called via the Jenkinsfile?

Reply:
Story book is now built once after the node modules are installed. previously we would run the chromaticqa tests (which builds storybook to a tmp dir) then we would run `deploy-storybook` which would build storybook again.

This avoids the double storybook build as we can get chromaticqa to use the already built storybook.